packages/fe-fpm-writer/test/unit/building-block.test.ts

```diff
@@ -168,6 +168,60 @@ describe('Building Blocks', () => {
         ).toThrowError(/Unable to parse xml view file/);
     });
 
+    describe.only('Generate with just ID and xml view without macros namespace', () => {
```

Comment:
I am sure the `only` is not supposed to stay :)

Reply:
Fixed.